### PR TITLE
DHFPROD-4719: Show warning if object or array datatype present.

### PR DESCRIPTION
**Pull request by timur-isangulov**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/query-export/query-export-modal/query-export-modal.test.tsx [hunk 1]
```diff
@@ -6,26 +6,16 @@ import QueryExportModal from './query-export-modal';
 describe('Query Export Component', () => {
 
   let columns = ['id', 'firstName', 'lastName', 'age']
+  let columnsNested = ['id', 'firstName', 'lastName', 'age', 'phoneNumber.work']
 
   test('Verify Query Export Modal Dialog renders', () => {
-    const { getByTestId, getByText } = render(<QueryExportModal exportModalVisibility={true} columns={columns} />);
+    const { getByTestId, getByText, queryByText } = render(<QueryExportModal exportModalVisibility={true} columns={columns} />);
+    expect(queryByText('export-warning')).not.toBeInTheDocument();
```
Comment (brucean52) [inline marklogic-data-hub-central/ui/src/components/query-export/query-export-modal/query-export-modal.test.tsx:13]: use toBeNull() for not existing

File: marklogic-data-hub-central/ui/src/components/query-export/query-export-modal/query-export-modal.test.tsx [hunk 2]
```diff
@@ -71,4 +61,9 @@ describe('Query Export Component', () => {
     expect(queryByText('Limited set of the first rows returned"')).not.toBeInTheDocument();
```
Comment (brucean52) [inline marklogic-data-hub-central/ui/src/components/query-export/query-export-modal/query-export-modal.test.tsx:61]: same here

File: marklogic-data-hub-central/ui/src/components/query-export/query-export-modal/query-export-modal.test.tsx (hunk 1 above)
Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/src/components/query-export/query-export-modal/query-export-modal.test.tsx]: Fixed. Thanks.

File: marklogic-data-hub-central/ui/src/components/query-export/query-export-modal/query-export-modal.test.tsx (hunk 2 above)
Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/src/components/query-export/query-export-modal/query-export-modal.test.tsx]: Fixed. Thanks.

**Pull request merged**